.github/workflows/build-images.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: 'Build Docker images'
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm/v7
+          push: true
+          tags: nutsfoundation/nuts-node:latest
```

Comment:
does this automatically create tags for each architecture? eg: `armv7-latest`

Reply:
All architectures are tagged under `latest`, see tag `arch-test` on [https://hub.docker.com/r/nutsfoundation/nuts-node/tags](https://hub.docker.com/r/nutsfoundation/nuts-node/tags). 

From [docker docs](https://docs.docker.com/docker-for-mac/multi-arch/): 
> Docker images can support multiple architectures, which means that a single image may contain variants for different architectures, and sometimes for different operating systems, such as Windows.
> 
> When running an image with multi-architecture support, docker will automatically select an image variant which matches your OS and architecture.